tests/inputs/input.DSHAPE_desc

```diff
@@ -8,11 +8,11 @@ NFP =   1
 Psi = 1.00000000
 
 # spectral resolution
-L_rad = 12
```

Comment:
why the changes to these resolutions?

Reply:
This change was needed so that `tests/inputs/input.DSHAPE` is consistent with `tests/inputs/input.DHSAPE_desc`. The VMEC convention is such that `MPOL = eq.M + 1`, so I decreased the resolution in the DESC input file to match the resolution of the VMEC input file.